data-prepper-plugins/aggregate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/aggregate/actions/CountAggregateActionConfig.java

```diff
@@ -17,21 +18,27 @@ public class CountAggregateActionConfig {
     public static final String DEFAULT_END_TIME_KEY = "aggr._end_time";
     public static final Set<String> validOutputFormats = new HashSet<>(Set.of(OutputFormat.OTEL_METRICS.toString(), OutputFormat.RAW.toString()));
 
+    @JsonPropertyDescription("Key used for storing the count. Default name is aggr._count.")
     @JsonProperty("count_key")
     String countKey = DEFAULT_COUNT_KEY;
 
+    @JsonPropertyDescription("Metric name to be used when otel format is used.")
     @JsonProperty("metric_name")
     String metricName = SUM_METRIC_NAME;
 
+    @JsonPropertyDescription("List of unique keys to count.")
     @JsonProperty("unique_keys")
     List<String> uniqueKeys = null;
 
+    @JsonPropertyDescription("Key used for storing the start time. Default name is aggr._start_time.")
     @JsonProperty("start_time_key")
     String startTimeKey = DEFAULT_START_TIME_KEY;
 
+    @JsonPropertyDescription("Key used for storing the end time. Default name is aggr._end_time.")
     @JsonProperty("end_time_key")
     String endTimeKey = DEFAULT_END_TIME_KEY;
 
+    @JsonPropertyDescription("Format of the aggregated event.otel_metrics: Default output format. Outputs in OTel metrics SUM type with count as value.raw - Generates a JSON object with the count_key field as a count value and the start_time_key field with aggregation start time as value.")
```

Comment:
This description is a little confusing. Why the combination of `event.otel_metrics`, and `value.raw`? Also there is a typo in OTel with caps

Reply:
This is directly copied from https://opensearch.org/docs/latest/data-prepper/pipelines/configuration/processors/aggregate/. But there is some ambiguity here due to formatting. I will fix it.